src/courses/advanced/10.md

```diff
@@ -84,12 +88,15 @@ Let's add the lint step:
 
```

Comment:
the note above this one is broken, I think cause we're using `Note` instead of `note`